main.py

```diff
@@ -221,19 +232,16 @@ async def votekick(update: Update, context: CallbackContext):
 	assert voter is not None
 	assert chat is not None
 
-	if tuser.id == 777000:
-		if (db.get_trusted(voter.id) or await is_admin(chat, voter)):
-			await update.message.reply_text(
-				"You can't votekick the channel…",
-				parse_mode=ParseMode.MARKDOWN_V2
-			)
-		else:
-			await update.message.delete()
-	elif not (db.get_trusted(voter.id) or await is_admin(chat, voter)):
+	if not (db.get_trusted(voter.id) or await is_admin(chat, voter)):
 		await update.message.reply_text(
 			'Only trusted users can votekick someone',
 			parse_mode=ParseMode.MARKDOWN_V2
 		)
+        elif tuser.id == 777000:

```

Comment:
ayo what are spaces doin in my tab-indented file

Reply:
also please restore this if/elif hunk to the original control flow, this does something slightly different than before